### PR TITLE
fix: changed the logout icon from a trash to that of a log out icon

### LINKED ISSUE #4111: [NV-2829] 🔓 Update logout icons and text when a users click on the avatar
### 📜 Description

We can have a different icon for the logout button

### 👟 Reproduction steps

1. Click on the profile icon at the top right
2. Small Icon with sign-out button

### 👍 Expected behavior

Please implement the changes according to [this Figma designs](https://www.figma.com/file/mxViLXxtC0PnB0ehXDcafw/Crafting-updates?type=design&node-id=742-1491&mode=design&t=zxBADQISFjfH2Yvb-0)

### 👎 Actual Behavior with Screenshots

* 

### Novu version

0\.18.0

### npm version

*No response*

### node version

*No response*

### 📃 Provide any additional context for the Bug.

* 

### 👀 Have you spent some time to check if this bug has been raised before?

* [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

* [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

<sub>From [SyncLinear.com](https://synclinear.com) | [NV-2829](https://linear.app/novu/issue/NV-2829/🔓-update-logout-icons-and-text-when-a-users-click-on-the-avatar)</sub>

### PR DESCRIPTION
**Pull request by maverox**
### What change does this PR introduce?

changed the logout icon to a more appropriate icon.

### Why was this change needed?

 Closes #4111

### Code Changes
- created a new component called Logout.tsx 
- pasted the code from Trash.tsx and replaced [svg](https://icons.getbootstrap.com/icons/box-arrow-right/) with the one mentioned in the issue
- and changed all the <Trash /> in HeaderNav.tsx with <Logout /> component
### Screenshot 

Screenshot 2023-10-20 165928
Screenshot 2023-10-20 175448

### CODE REVIEW

File: apps/web/src/design-system/icons/index.ts [hunk 1]
```diff
@@ -84,6 +85,7 @@ export { Cloud } from './general/Cloud';
 export { Condition } from './general/Condition';
 export { RemoveCondition } from './general/RemoveCondition';
 export { Warning } from './general/Warning';
+export { Logout } from './general/Logout';
```
Comment (scopsy) [inline apps/web/src/design-system/icons/index.ts:88]: Is this a duplicate export here?

Comment (maverox) [reply apps/web/src/design-system/icons/index.ts:88]: Oh yes! thanks for pointing it out, working on it

Comment (maverox) [reply apps/web/src/design-system/icons/index.ts:87]: edited the file, can you please check it?

**Comment by maverox** [comment]
> Thank you so much for your contribution @maverox!
> 
> I have noticed that the icon for the logout is not the proper icon, and the alignment looks off. (Attaching a screenshot) CleanShot 2023-10-03 at 17 57 53@2x (the log out icon is shifted to the left compared to the original icon)
> 
> ```
> <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none"><mask id="a" width="20" height="20" x="0" y="0" maskUnits="userSpaceOnUse" style="mask-type:alpha"><path fill="#D9D9D9" d="M0 0h20v20H0z"/></mask><g mask="url(#a)"><path fill="#fff" d="M4.5 17c-.412 0-.766-.147-1.06-.44A1.445 1.445 0 0 1 3 15.5v-11c0-.412.147-.766.44-1.06.294-.293.648-.44 1.06-.44H10v1.5H4.5v11H10V17H4.5Zm9-3.5-1.063-1.063 1.688-1.687H8v-1.5h6.125l-1.688-1.688L13.5 6.5 17 10l-3.5 3.5Z"/></g></svg>
> ```
> 
> The SVG is taken as an export from figma on this link: https://www.figma.com/file/mxViLXxtC0PnB0ehXDcafw/Crafting-updates?node-id=742%3A1859&mode=dev (right clicking and then copy as SVG)

actually, i copied the svg from the bootstrap link from the issue. I am working on aligning it. thanks for this extensive support and guiding me.

**Comment by maverox** [comment]
@scopsy I am seeming to have a problem setting up local environment it says cross-env not recognised as a batch command and also i tried setting up in gitpod but still some issues persist
ports are not exposed and many more such issues.
I am really sorry to make you wait!

**Comment by scopsy** [comment]
@maverox have you ran `npm run setup:project` from the root of the repo?

**Comment by maverox** [comment]
this cross-env error only occurs when i am in local environment and not in gitpod i assume that it's preinstalled or something there

**Comment by scopsy** [comment]
@maverox very strange :( Did `pnpm install` finished successfully from root of monorepo?

**Comment by maverox** [comment]
No, pnpm install fails everytime i tried!


Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ainouzgali** [comment]
Hi @maverox. Could you please open a thread on our support channel on discord? It will be so helpful for us. It will be easier to help you from there

**Comment by ainouzgali** [comment]
@maverox please let us know if you are still interested in working on this :) we will be happy to help.

**Comment by maverox** [comment]
> @maverox please let us know if you are still interested in working on this :) we will be happy to help.

yes i am interested in learning and getting through these setbacks, actually i was unavailable as i had my hands full with college work.

**Comment by maverox** [comment]
can you please review this PR now i made the necessary changed as advised!
Please guide me as to what to do next regarding this contribution.

**Comment by p-fernandez** [comment]
Looks good.
Screenshot 2023-10-21 at 13 05 06

**Review by p-fernandez — APPROVED** [review]
🌟

**Comment by maverox** [comment]
> 🌟

Can you Please tell me what's left before a successful merge. I am curious!😁

**Pull request merged**